lib/msal-angular/src/packageMetadata.ts

```diff
@@ -1,3 +1,3 @@
 /* eslint-disable header/header */
 export const name = "@azure/msal-angular";
-export const version = "2.5.3";
+export const version = "3.0.0-alpha";
```

Comment:
Should this be alpha.0? And do we need to update the package.json version to match, or will that be done at release time?